Update CSES Flight Routes Check.mdx (#996)
cpinitiative/usaco-guide: solutions/CSES Flight Routes Check.mdx

# Main Idea

> The theorem used here is, if one vertex can reach all, and it can BE reached by all, then every vertex in this graph is interconnected to each other.

Reviewer: ```suggestion
The theorem used here is that if one vertex can both reach and **be reached** by all others, then every vertex in this graph can reach all others.
```
